content/docs/1.5/scalers/rabbitmq-queue.md

```diff
@@ -15,29 +15,38 @@ This specification describes the `rabbitmq` trigger for RabbitMQ Queue.
 triggers:
 - type: rabbitmq
   metadata:
-    host: RabbitMqHost
-    queueLength: '20' # Optional. Queue length target for HPA. Default: 20 messages
+    queueLength: '20'
     queueName: testqueue
     includeUnacked: 'true' # Optional, use unacked + ready messages count
-    apiHost: RabbitApiHost # Optional HTTP management API endpoint
+  authenticationRef:
+    name: keda-trigger-auth-rabbitmq-conn
 ```
 
-The `host` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.  The resolved host should follow a format like `amqp://guest:password@localhost:5672/vhost`
+**Parameter list:**
```

Comment:
You should still list `host` & `apiHost` here since they are also parameters which you can specify in the YAML spec

Reply:
Done